app/Http/Controllers/Api/StatuslabelsController.php

```diff
@@ -182,6 +182,7 @@ public function getAssetCountByStatuslabel()
 
                 if ($statuslabel->color!='') {
                     $colors_array[] = $statuslabel->color;
+                    $default_color_count++;
```

Comment:
If we're going to do this on both sides of the if/else, we should probably just set it above the if/else to avoid repeating code, no?

Reply:
That's why you are the boss, boss. You are totally correct, (and I'm a little embarrassed 😳 to not think of that)... Making the change!!